render_silhouette_regmarks.py

```diff
@@ -75,54 +76,56 @@ def effect(self):
 			inkex.base.InkscapeExtension.msg(gettext("[INFO]: regmark to regmark spacing Y ")+str(reg_length))
 
 		# Create a new layer
-		layer = self.svg.add(inkex.Layer.new(self.LAYERNAME))
+		layer = self.svg.add(Layer.new(LAYERNAME))
 	
 		# Create square in top left corner
-		layer.append(self.drawRect((self.REG_SQUARE_MM,self.REG_SQUARE_MM), (reg_origin_X,reg_origin_Y), 'TopLeft'))
+		layer.append(self.drawRect((REG_SQUARE_MM,REG_SQUARE_MM), (reg_origin_X,reg_origin_Y), 'TopLeft'))
 		
 		# Create group for top right corner
-		topRight = inkex.Group(id = 'TopRight', style = 'fill: black;')
+		topRight = Group(id = 'TopRight')
 		# Create horizontal and vertical lines in group
 		top_right_reg_origin_x = reg_origin_X+reg_width
-		topRight.append(self.drawLine((top_right_reg_origin_x-self.REG_LINE_MM,reg_origin_Y), (top_right_reg_origin_x,reg_origin_Y), 'Horizontal'))
-		topRight.append(self.drawLine((top_right_reg_origin_x,reg_origin_Y), (top_right_reg_origin_x,reg_origin_Y + self.REG_LINE_MM), 'Vertical'))
+		topRight.append(self.drawLine((top_right_reg_origin_x-REG_LINE_MM,reg_origin_Y), (top_right_reg_origin_x,reg_origin_Y), 'Horizontal'))
+		topRight.append(self.drawLine((top_right_reg_origin_x,reg_origin_Y), (top_right_reg_origin_x,reg_origin_Y + REG_LINE_MM), 'Vertical'))
 		layer.append(topRight)
 		
 		# Create group for top right corner
-		bottomLeft = inkex.Group(id = 'BottomLeft', style = 'fill: black;')
+		bottomLeft = Group(id = 'BottomLeft')
 		# Create horizontal and vertical lines in group
 		top_right_reg_origin_y = reg_origin_Y+reg_length
-		bottomLeft.append(self.drawLine((reg_origin_X,top_right_reg_origin_y), (reg_origin_X+self.REG_LINE_MM,top_right_reg_origin_y), 'Horizontal'))
-		bottomLeft.append(self.drawLine((reg_origin_X,top_right_reg_origin_y), (reg_origin_X,top_right_reg_origin_y - self.REG_LINE_MM), 'Vertical'))
+		bottomLeft.append(self.drawLine((reg_origin_X,top_right_reg_origin_y), (reg_origin_X+REG_LINE_MM,top_right_reg_origin_y), 'Horizontal'))
+		bottomLeft.append(self.drawLine((reg_origin_X,top_right_reg_origin_y), (reg_origin_X,top_right_reg_origin_y - REG_LINE_MM), 'Vertical'))
 		layer.append(bottomLeft)
 		
 		# Keepout Marker #
+		# Not directly part of Silhouette registration marker
+		# Instead it's a visual indicator to the user to avoid putting any design within this area
```

Comment:
thanks for this explanation on providing visual guidance to the user. 
would you agree to switch to something softer i.e.  similar to what is used in 
the preexisting template here https://github.com/fablabnbg/inkscape-silhouette/blob/main/templates/silhouette-cameo-registration-marks-a4.svg
* checkerboard: true https://github.com/fablabnbg/inkscape-silhouette/blob/af3c9019e948aa56c9a357263eec45a35df18f20/templates/silhouette-cameo-registration-marks-a4.svg#L42
* path with fill: `#ffffff` https://github.com/fablabnbg/inkscape-silhouette/blob/af3c9019e948aa56c9a357263eec45a35df18f20/templates/silhouette-cameo-registration-marks-a4.svg#L137-L142